### PR TITLE
fix(http): The org parameter takes either the ID or Name interchangeably

### LINKED ISSUE #13906: InfluxDB 2.0: The "org" query param should be same for /query and /write  [area/api]
The `/query` and the `/write` are critical parts of API 2.0 and should be friendly for users.

The `/write` endpoint has defined `org` parameter as:

```yaml
- in: query
   name: org
   description: specifies the destination organization for writes
   required: true
   schema:
      type: string
      description: all points within batch are written to this organization.
```

The `org` parameter could be `Organization.name` or `Organization.id` - [see sources](https://github.com/influxdata/influxdb/blob/master/http/write_handler.go#L137).

But the `/query` endpoint has define `org` and `orgID` parameter:
```yaml
- in: query
   name: org
   description: specifies the name of the organization executing the query; if both orgID and org are specified, orgID takes precedence.
   schema:
      type: string
- in: query
   name: orgID
   description: specifies the ID of the organization executing the query; if both orgID and org are specified, orgID takes precedence.
   schema:
      type: string
```

For consistency in API should be better that  the `/query` endpoint will be same as `/write` endpoint.

It will be also useful for clients configuration. For example both of this [Spring actuator configuration](https://github.com/bonitoo-io/influxdb-client-java/tree/master/spring#influxdb2-auto-configuration) will be valid:

```yaml
spring.influx2.url=http://localhost:9999
spring.influx2.org=my-org
spring.influx2.bucket=my-bucket
spring.influx2.token=my-token-123
spring.influx2.username=my-user
```

```yaml
spring.influx2.url=http://localhost:9999
spring.influx2.org=03d2ebc848c1e000
spring.influx2.bucket=my-bucket
spring.influx2.token=my-token
spring.influx2.username=my-user
```

Maybe it is also related to https://github.com/influxdata/influxdb-client-js/pull/115#issuecomment-490310352 from @timhallinflux

### PR DESCRIPTION
**Pull request by bednar**
Closes #13906

The org parameter could be `Organization.name` or `Organization.id` for '/write' and '/query'.

### CODE REVIEW

File: http/requests.go [hunk 1]
```diff
@@ -12,21 +12,37 @@ const (
...
+			o, err := svc.FindOrganization(ctx, platform.OrganizationFilter{ID: id})
+			if err == nil {
+				return o, err
+			} else if platform.ErrorCode(err) != platform.ENotFound {
+				return nil, err
+			}
+		}
+
+		o, err := svc.FindOrganization(ctx, platform.OrganizationFilter{Name: &organization})
+		if err != nil {
```
Comment (kelwang) [inline http/requests.go]: I think as long as the IDFromString returns no error, we can assure it is a qualified id, and user want to use orgID. So  we should return "not found error" immediately, without keep doing another search.

It's simpler just doing this
```go
if organization := r.URL.Query().Get(Org); organization != "" {
		if id, err := platform.IDFromString(organization); err == nil {
                         filter.ID = id
                }else{
                         filter.Name = &organization
                }
}
if reqID := r.URL.Query().Get(OrgID); reqID != "" {
		filter.ID, err = platform.IDFromString(reqID)
		if err != nil {
			return nil, err
		}
	}
```

File: http/requests.go [hunk 2]
```diff
@@ -12,21 +12,37 @@ const (
 	OrgName = "org"
```
Comment (kelwang) [inline http/requests.go]: remove this line, if it's not being used everywhere

File: http/requests.go (hunk 1 above)
Comment (bednar) [reply http/requests.go]: Fixed

File: http/requests.go (hunk 2 above)
Comment (bednar) [reply http/requests.go]: Fixed

**Pull request merged**